packages/sdk-codegen/src/typescript.gen.ts

```diff
@@ -510,11 +529,13 @@ export interface IDictionary<T> {
             name: `${map.name}[]`,
           }
         case 'HashType':
+          this.rtlNeeds.add('IDictionary')
```

Comment:
thank goodness for `Set`s

Reply:
Yes!

Reply:
We track `yarn.lock` in other repositories so that's why I've been doing it here. The spec generation could be fine to ignore but I just prettify it and commit it again without doing a very deep compare. I wiped just to make sure the 21.4 reference release was used for the latest saved spec.